internal/service/anycast/model_proto_anycast_config.go

```diff
@@ -0,0 +1,181 @@
+	AnycastIpv6Address types.String      `tfsdk:"anycast_ipv6_address"`
+	CreatedAt          timetypes.RFC3339 `tfsdk:"created_at"`
+	Description        types.String      `tfsdk:"description"`
+	Id                 types.Int64       `tfsdk:"id"`
+	IsConfigured       types.Bool        `tfsdk:"is_configured"`
+	Name               types.String      `tfsdk:"name"`
+	OnpremHosts        types.List        `tfsdk:"onprem_hosts"`
+	RuntimeStatus      types.String      `tfsdk:"runtime_status"`
+	Service            types.String      `tfsdk:"service"`
+	Tags               types.Map         `tfsdk:"tags"`
+	UpdatedAt          timetypes.RFC3339 `tfsdk:"updated_at"`
+}
+
+var ProtoAnycastConfigAttrTypes = map[string]attr.Type{
+	"account_id":           types.Int64Type,
+	"anycast_ip_address":   types.StringType,
+	"anycast_ipv6_address": types.StringType,
+	"created_at":           timetypes.RFC3339Type{},
+	"description":          types.StringType,
+	"id":                   types.Int64Type,
+	"is_configured":        types.BoolType,
+	"name":                 types.StringType,
+	"onprem_hosts":         types.ListType{ElemType: types.ObjectType{AttrTypes: ProtoOnpremHostRefAttrTypes}},
+	"runtime_status":       types.StringType,
+	"service":              types.StringType,
+	"tags":                 types.MapType{ElemType: types.StringType},
+	"updated_at":           timetypes.RFC3339Type{},
+}
+
+var ProtoAnycastConfigResourceSchemaAttributes = map[string]schema.Attribute{
+	"account_id": schema.Int64Attribute{
+		Computed:            true,
+		MarkdownDescription: "The account identifier.",
+	},
+	"anycast_ip_address": schema.StringAttribute{
+		Required: true,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIfConfigured(),
+		},
+		MarkdownDescription: "IPv4 address of the host in string format.",
+	},
+	"anycast_ipv6_address": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "IPv6 address of the host in string format",
+	},
+	"created_at": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "Time when the object has been created.",
+	},
+	"description": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "The description for the address object. May contain 0 to 1024 characters. Can include UTF-8.",
+	},
+	"id": schema.Int64Attribute{
+		Computed: true,
+		PlanModifiers: []planmodifier.Int64{
+			int64planmodifier.UseStateForUnknown(),
+		},
+		MarkdownDescription: "The resource identifier.",
+	},
+	"is_configured": schema.BoolAttribute{
+		Computed: true,
+	},
+	"name": schema.StringAttribute{
+		Required:            true,
+		MarkdownDescription: `The name of the anycast configuration.`,
+	},
+	"onprem_hosts": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: ProtoOnpremHostRefResourceSchemaAttributes,
+		},
+		Optional: true,
```

Comment:
as discussed wasn't this supposed to be `Computed`?

Reply:
done